src/styles/blocks/card.scss

```diff
@@ -0,0 +1,104 @@
+
+.cards {
+  @extend %flex__justify-content;
+  flex-wrap: wrap;
+  @extend %reset-margin;
+  padding: 50px 40px;
+  &__card {
+    box-sizing: border-box;
+    margin: 0 48px 46px 0 ;
+    &:nth-child(4n) {
+      margin-right: 0;
+    }
+  }
+}
+
```

Comment:
Everything is correct now , just need 1 more fix
`cards` and `card` are 2 diffrent blocks so transfer this `.cards` styles to `cards.scss`